cfgov/v1/models/snippets.py

```diff
@@ -0,0 +1,27 @@
+from django.db import models
+
+from wagtail.wagtailsnippets.models import register_snippet
+from wagtail.wagtailcore.fields import RichTextField, StreamField
+from wagtail.wagtailadmin.edit_handlers import FieldPanel, StreamFieldPanel
+
+from . import molecules
+
+
+@register_snippet
+class Contact(models.Model):
+    heading = models.CharField(max_length=22, blank=False)
```

Comment:
Heading is what we'll use to find unique contacts, filling the slug role, though we could sit down and have a discussion on if we want to create a base snippet class for our site which contains a slug.